pkg/store/driver/txn/txn_driver.go

```diff
@@ -78,6 +81,10 @@ func (txn *tikvTxn) CacheTableInfo(id int64, info *model.TableInfo) {
 }
 
 func (txn *tikvTxn) LockKeys(ctx context.Context, lockCtx *kv.LockCtx, keysInput ...kv.Key) error {
+	if intest.InTest {
+		txn.isCommitterWorking.Store(true)
```

Comment:
Why set `isCommitterWorking` when `InTest`?  The `InTest` is a little confusing.